components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/validators/TokenValidationHandler.java

```diff
@@ -473,13 +487,17 @@ private OAuth2IntrospectionResponseDTO validateAccessToken(OAuth2TokenValidation
             return buildIntrospectionErrorResponse("Invalid access delegation");
         }
 
-        // Validate scopes.
+        // Validate scopes at app level.
         if (!tokenValidator.validateScope(messageContext)) {
             // This is redundant. But sake of readability.
             introResp.setActive(false);
+            if (log.isDebugEnabled()) {
+                log.debug("Scope validation has failed at app level");
```

Comment:
add fullstop at the end

Reply:
added in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1468/commits/a1fc7ccf713f226608dc667d1bdcf13cd22ad83d